Ballerina.lock

```diff
@@ -1,4 +1,329 @@
 org_name = "ballerinax"
 version = "0.1.0"
 lockfile_version = "1.0.0"
-ballerina_version = "1.2.2"
+ballerina_version = "2.0.0-Preview1-M6"
```

Comment:
```suggestion
ballerina_version = "slp1"
```

Reply:
This is an auto-generated file. once you build with swan lake previews, this will auto correct